### ISSUE TITLE
Bug: padding is missing below

### ISSUE THREAD

**Issue by compulim** [Microsoft/BotFramework-WebChat#1261]
@seaen

![image](https://user-images.githubusercontent.com/1622400/47758691-5de89300-dc69-11e8-9eb7-8520954def94.png)

```html
<html lang="en-US">
  <head>
    <title>Web Chat: Full-featured bundle</title>
    <script src="https://cdn.botframework.com/botframework-webchat/preview/botchat.js"></script>
    <style>
      html, body { height: 100% }
      body { margin: 0 }

      #webchat,
      #webchat > * {
        height: 100%;
        width: 100%;
      }
    </style>
  </head>
  <body>
    <div id="webchat"></div>
    <script>
      window.WebChat.renderWebChat({
        directLine: window.WebChat.createDirectLine({ secret: 'YOUR_SECRET_HERE' })
      }, document.getElementById('webchat'));

      document.querySelector('#webchat > *').focus();
    </script>
  </body>
</html>
```

**Comment by compulim** [comment]
The HTML file is missing `<!DOCTYPE html>`. Thus, Chrome/Edge does not treat this as HTML5 and it break the UI. E.g. `<form>` in HTML5 is margin-less, while in HTML4, it has `margin-bottom: 1em`.

**Comment by compulim** [comment]
Will fix.

**Comment by compulim** [comment]
Fixed in #1260.
